packages/react/tsconfig.json

```diff
@@ -8,24 +8,12 @@
     "outDir": "./lib",
     "moduleResolution": "node",
     "skipLibCheck": true,
-    "lib": [
-      "es2018",
-      "es2020",
-      "dom"
-    ],
+    "lib": ["es2018", "es2020", "dom"],
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
     "jsx": "react",
     "downlevelIteration": true
   },
-  "include": [
-    "src"
-  ],
-  "exclude": [
-    "node_modules",
-    "**/*.spec.ts",
-    "**/*.spec.tsx",
-    "**/*.test.ts",
-    "**/*.test.tsx"
-  ]
+  "include": ["src", "typings"],
+  "exclude": ["node_modules"]
```

Comment:
isn't it part of build now?

Reply:
I'll add tsconfig.prod.ts which will be used for building everywhere